cli/src/progress.rs

```diff
@@ -80,14 +81,18 @@ where
 {
     use std::ops::Deref;
     let mut template_str = String::new();
-    template_str.push_str(&format!("{} ", LOG_PREFIX_INFO.deref()));
```

Comment:
this was the specific lint: https://rust-lang.github.io/rust-clippy/master/index.html#format_push_string
not sure if this is the best way to fix it

Reply:
This lint is problematic because of the way the proposed fix introduces fallibility (see [discussion and restriction](https://github.com/rust-lang/rust-clippy/issues/9077)). Still, this change should be fine here.